### PR TITLE
Load tickets blocks only on supported post types

### PR DESCRIPTION
**Pull request by mitogh**
- Add support for tickets blocks only when tickets are supported
- Load assets on supported types
- Remove duplication code shared among editor classes

### CODE REVIEW

File: plugins/tickets/src/Tribe/Editor.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	 * @param array  $template  Array of all the templates used by default
+	 * @param string $post_type The current post type
+	 *
+	 * @return array
+	 */
+	public function add_tickets_block_in_editor() {
+		foreach ( $this->supported_types() as $post_type ) {
+			$post_type_object = get_post_type_object( $post_type );
+			if ( ! $post_type_object ) {
+				var_dump( $post_type );
```
Comment (bordoni) [inline plugins/tickets/src/Tribe/Editor.php]: Dump?

Comment (mitogh) [reply plugins/tickets/src/Tribe/Editor.php]: Doh, thanks will fix ASAP

**Comment by mitogh** [comment]
Change has been applied, thanks.

File: plugins/tickets/src/modules/blocks/tickets/index.js [hunk 2]
```diff
@@ -7,8 +7,9 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import { BlockIcon } from '@moderntribe/common/elements';
-import { TicketActive } from '@moderntribe/tickets/icons';
 import DisabledTickets from './DisabledTickets';
+import TicketIcon from './TicketIcon';
+import { ActionDashboard } from '@moderntribe/tickets/elements';
```
Comment (paulmskim) [inline plugins/tickets/src/modules/blocks/tickets/index.js]: unused import?

Comment (mitogh) [reply plugins/tickets/src/modules/blocks/tickets/index.js]: Yep removing on the next PR where Container / Template are done 👍

**Pull request merged**